Description: Removes SSE and SSE2 support
 This makes sure it builds compatible binaries also on 
 machines which have sse/sse2. Fixes the SIGILL bug.
 .

--- blender-2.63a.orig/build_files/cmake/macros.cmake
+++ blender-2.63a/build_files/cmake/macros.cmake
@@ -359,33 +359,9 @@ macro(TEST_SSE_SUPPORT
 
 	set(CMAKE_REQUIRED_FLAGS "${${_sse_flags}} ${${_sse2_flags}}")
 
-	if(NOT DEFINED SUPPORT_SSE_BUILD)
-		# result cached
-		check_c_source_runs("
-			#include <xmmintrin.h>
-			int main(void) { __m128 v = _mm_setzero_ps(); return 0; }"
-		SUPPORT_SSE_BUILD)
-
-		if(SUPPORT_SSE_BUILD)
-			message(STATUS "SSE Support: detected.")
-		else()
-			message(STATUS "SSE Support: missing.")
-		endif()
-	endif()
+	message(STATUS "SSE Support: unused.")
 
-	if(NOT DEFINED SUPPORT_SSE2_BUILD)
-		# result cached
-		check_c_source_runs("
-			#include <emmintrin.h>
-			int main(void) { __m128d v = _mm_setzero_pd(); return 0; }"
-		SUPPORT_SSE2_BUILD)
-
-		if(SUPPORT_SSE2_BUILD)
-			message(STATUS "SSE2 Support: detected.")
-		else()
-			message(STATUS "SSE2 Support: missing.")
-		endif()
-	endif()
+	message(STATUS "SSE2 Support: unused.")
 
 	unset(CMAKE_REQUIRED_FLAGS)
 endmacro()
